media-sound/teamspeak-server-bin/teamspeak-server-bin-3.1.1.ebuild

```diff
@@ -0,0 +1,90 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+inherit multilib systemd user
+
+DESCRIPTION="Crystal Clear Cross-Platform Voice Communication Server"
+HOMEPAGE="https://www.teamspeak.com/"
+SRC_URI="
+	amd64? ( http://ftp.4players.de/pub/hosted/ts3/releases/${PV}/teamspeak3-server_linux_amd64-${PV}.tar.bz2 )
+	x86? ( http://ftp.4players.de/pub/hosted/ts3/releases/${PV}/teamspeak3-server_linux_x86-${PV}.tar.bz2 )"
+
+SLOT="0"
+LICENSE="teamspeak3"
+IUSE="doc tsdns"
+KEYWORDS="~amd64 ~x86"
+
+RESTRICT="installsources fetch mirror strip"
+
+S="${WORKDIR}/teamspeak3-server_linux_${ARCH}"
+
+QA_PREBUILT="opt/*"
+
+pkg_nofetch() {
+	elog "Please download ${A}"
+	elog "from ${HOMEPAGE}downloads and place this"
+	elog "file in ${DISTDIR}"
```

Comment:
`DISTDIR` is forbidden in `pkg_nofetch`.

Reply:
Also fixed this.